cdap-ui/app/cdap/components/Cloud/Profiles/CreateView/index.js

```diff
@@ -77,9 +85,25 @@ export default class ProfilesCreateView extends Component {
         })
       }
     };
-    console.log(`Coming soon. This is the state right now:
-      ${JSON.stringify(jsonBody, null, 2)}
-    `);
+    MyProfileApi
+      .create({
+        namespace: getCurrentNamespace(),
+        profile: this.state.profileName
+      }, jsonBody)
+      .subscribe(
+        () => {
+          this.setState({
+            redirectToNamespace: true,
+            loading: true
+          });
+        },
+        err => {
+          this.setState({
+            loading: false,
+            error: JSON.stringify(err, null, 2)
```

Comment:
nit: you're not showing this error anywhere yet

Reply:
Yes. We need to add the error handling capability. Right now just adding to the state. Will have the error handling in another PR. Right now trying to get in the functionality for the demo.